src/core/Timepicker/base.ts

```diff
@@ -0,0 +1,3 @@
+import { TimePickerProps } from '@material-ui/pickers';
+
+export type BaseTimePickerProps = Omit<TimePickerProps, 'value'>;
```

Comment:
We should be able to use the basic props provided by KeyboardTimePicker, there's no requirement to omit any